card/mod/account/set/right/salt.rb

```diff
@@ -1,5 +1,5 @@
 include All::Permissions::Accounts
 
-view :raw do |_args|
+view :raw do
```

Comment:
this is a case where raw makes sense, imo.  The raw view is effectively an override of the card content (which needs to be kept private in this case)